spec/spec_helper.rb

```diff
@@ -32,14 +31,14 @@
     AR2DTO::Config.reset! if Object.const_defined?("AR2DTO::Config")
   end
 
-  config.before(:suite) do
-    DatabaseCleaner.strategy = :transaction
-    DatabaseCleaner.clean_with(:truncation)
-  end
-
   config.around(:each) do |example|
-    DatabaseCleaner.cleaning do
-      example.run
-    end
+    User.delete_all
+    Person.delete_all
+    Car.delete_all
+    Shop::LineItem.delete_all
+    Shop::Order.delete_all
+    Other.delete_all
```

Comment:
hmm I don't like having to remember to maintain this. Also we might be leaving things in some cases like failure or interruptions. Maybe we can use a fork of database cleaner or something that fixes the issue?

Reply:
Sounds good. Maybe I will wait for DatabaseCleaner to release a new version which seems they are working on.